backend/performance/locustfile.py

```diff
@@ -0,0 +1,92 @@
+from locust import HttpUser, task, between
+
+TOKEN = "eyJhbGciOiJSUzI1NiIsInR5cCIgOiAiSldUIiwia2lkIiA6ICJxUWlFWDB2T2Z1SlBuWUw4MWo0Q2tDOHVPdEJ1aFZvM0xBd2ppczZWbHRzIn0.eyJleHAiOjE3NDQ2NzI0MjksImlhdCI6MTc0NDY3MjEyOSwiYXV0aF90aW1lIjoxNzQ0NjY5NjUyLCJqdGkiOiI5YTEzN2ViYi0zZTdlLTRhM2ItOTYxMC0yZmVlNGU3NDI1OGUiLCJpc3MiOiJodHRwczovL2Rldi5sb2dpbnByb3h5Lmdvdi5iYy5jYS9hdXRoL3JlYWxtcy9zdGFuZGFyZCIsImF1ZCI6Imxvdy1jYXJib24tZnVlbC1zdGFuZGFyZC01MTQ3Iiwic3ViIjoiZDAyNTMyOTgyOTgzNGE5Nzk5OTgyYWI1ZjdiYzg2NGJAYmNlaWRidXNpbmVzcyIsInR5cCI6IkJlYXJlciIsImF6cCI6Imxvdy1jYXJib24tZnVlbC1zdGFuZGFyZC01MTQ3Iiwibm9uY2UiOiIyYjg1ZjI3My01OGUyLTQxNmItODgzNi04MDQ4ZjA4Y2FmMzEiLCJzaWQiOiI5ZTk2MjBjYy0yN2QxLTQzNTYtOTk5Yi03MDdmMmIzYjA1MDciLCJzY29wZSI6Im9wZW5pZCBpZGlyIGJjZWlkYnVzaW5lc3MgZW1haWwgcHJvZmlsZSIsImJjZWlkX2J1c2luZXNzX2d1aWQiOiI1NEE3MzIzNUI2MEU0ODcwOTUzMEMxNzAwMEFBNUY0MSIsImJjZWlkX2J1c2luZXNzX25hbWUiOiJUZWFtLCBaZWxkYSIsImJjZWlkX3VzZXJfZ3VpZCI6IkQwMjUzMjk4Mjk4MzRBOTc5OTk4MkFCNUY3QkM4NjRCIiwiYmNlaWRfdXNlcm5hbWUiOiJsY2ZzMSIsImVtYWlsX3ZlcmlmaWVkIjpmYWxzZSwicHJlZmVycmVkX3VzZXJuYW1lIjoiZDAyNTMyOTgyOTgzNGE5Nzk5OTgyYWI1ZjdiYzg2NGJAYmNlaWRidXNpbmVzcyIsImRpc3BsYXlfbmFtZSI6IkphbmUgRG9lIiwiZ2l2ZW5fbmFtZSI6IkphbmUgRG9lIiwiaWRlbnRpdHlfcHJvdmlkZXIiOiJiY2VpZGJ1c2luZXNzIiwibmFtZSI6IkphbmUgRG9lIiwic2Vzc2lvbl9zdGF0ZSI6IjllOTYyMGNjLTI3ZDEtNDM1Ni05OTliLTcwN2YyYjNiMDUwNyIsImZhbWlseV9uYW1lIjoiIiwiZW1haWwiOiJ0ZnJzQGdvdi5iYy5jYSJ9.gHs1YCtv2DWEFQFPA6ddecwA2X5eaINLPvp_ypS0kRmrLb2UdPVGBmdD9s2QLPH719r_szqHuvl_gjPiWsHyMTU_UM2yzFixFSVFYhNcJs8TREBs6v6H-knHmCH3Y2beFbmF8TKBD0jSg7lmnfkqxlBoY36E-t6U-pMmUYvW_JUk5f_dUpsKpzmwuoh3hMJmGig-z2kdRxbINBXgYDDifnWXqXxHAFbP0TqOQwjLcG43VHXHibAHB-1_2Q0UgwtxZb1sCNx6XwoRe1fubCn4V2lLsnpbnKjASHsMF01shDOcWo-RaHzGnFibFIQJiInp6aMZckb1i6f78nUroUe4KQ"
```

Comment:
lets hard remove this from the git history. just a blank string plz

Reply:
Done!